lib/routing/sfRoute.class.php

```diff
@@ -264,7 +265,13 @@ public function generate($params, $context = array(), $absolute = false)
 
   static private function generateCompareVarsByStrlen($a, $b)
   {
-    return strlen($a) < strlen($b);
+    if (strlen($a) < strlen($b)) {
+        return 1;
+    }
+    if (strlen($a) > strlen($b)) {
+        return -1;
+    }
+    return 0;
```

Comment:
```suggestion
    return strlen($a) <=> strlen($b);
```

For fun. :) https://www.php.net/manual/en/language.operators.comparison.php

Reply:
I've used the comparison so little that I even forgot it exists. 😅 Great simplification, thanks!